text/trace/0192-messaging-semantic-conventions-spec.md

```diff
@@ -0,0 +1,659 @@
+
+* [Motivation](#motivation)
+* [Proposed stable messaging semantic conventions for tracing](#proposed-stable-messaging-semantic-conventions-for-tracing)
+  - [Terminology](#terminology)
+  - [Stages of producing and consuming messages](#stages-of-producing-and-consuming-messages)
+  - [Context propagation](#context-propagation)
+  - [Trace structure](#trace-structure)
+  - [Span names, kinds, and attributes](#span-names-kinds-and-attributes)
+  - [System-specific extensions](#system-specific-extensions)
+  - [Examples](#examples)
+* [Future possibilities](#future-possibilities)
+  - [Transport context propagation](#transport-context-propagation)
+  - [Standards for context propagation](#standards-for-context-propagation)
+
+## Motivation
+
+This OTEP serves as a foundation for a first stable version of [messaging semantic conventions for tracing](https://github.com/open-telemetry/opentelemetry-specification/blob/a1a8676a43dce6a4e447f65518aef8e98784306c/specification/trace/semantic_conventions/messaging.md).
+It aims to define clear, consistent, and extensible conventions, it also
+describes reasons and motivations that led to the formulation of those
+conventions. The conventions comprise areas such as context propagation, trace
+structure, names, and attributes.
+
+After this OTEP is merged, the changes it proposes will be merged into the
+messaging semantic conventions, which will subsequently be declared
+[stable](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable).
+
+## Proposed stable messaging semantic conventions for tracing
+
+### Terminology
+
+The terminology used in this document is based on the [CloudEvents specification](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md).
+CloudEvents is hosted by the CNCF and provides a specification for describing
+event data in common formats to provide interoperability across services,
+platforms and systems.
+
+#### Message
+
+A "message" is a transport envelope for the transfer of information. The
+information is a combination of a payload and metadata. Metadata can be
+directed at consumers or intermediaries on the message path. Messages are
+transferred via one or more intermediaries.  Messages are uniquely
+identifiable.
+
+In the strict sense, a _message_ is a payload that is sent to a specific
+destination, whereas an _event_ is a signal emitted by a component upon
+reaching a given state. This document is agnostic of those differences and uses
+the term "message" in a wider sense to cover both concepts.
+
+#### Producer
+
+The "producer" is a specific instance, process or device that creates and
+publishes a message. "Publishing" is the process of sending a message or batch
+to the intermediary.
+
+#### Consumer
+
+A "consumer" receives the message and acts upon it. It uses the context and
+data to execute some logic, which might lead to the occurrence of new messages.
```

Comment:
It's probably the "creation span context" that is used to link a producer with a consumer. But I think we can remove this from here, since it's just defining terminology and setting the stage. Also the `creation context` term is not defined yet.

Reply:
Addressed in: [5fa715d](https://github.com/open-telemetry/oteps/pull/192/commits/5fa715dcc6a4e6adafce7fc4032d7f52c24fd31a)